ambry-server/src/main/java/com.github.ambry.server/ErrorMapping.java

```diff
@@ -35,6 +35,7 @@
     tempMap.put(StoreErrorCodes.ID_Not_Found, ServerErrorCode.Blob_Not_Found);
     tempMap.put(StoreErrorCodes.TTL_Expired, ServerErrorCode.Blob_Expired);
     tempMap.put(StoreErrorCodes.Already_Exist, ServerErrorCode.Blob_Already_Exists);
+    tempMap.put(StoreErrorCodes.Authorization_Failure, ServerErrorCode.Blob_Not_Found);
```

Comment:
I suppose this is okay for now, but should we eventually add a new `ServerErrorCode` for auth failure and add handling for it in the router? Right now, this could make the router send a request to every replica.

Reply:
There also seems to be a `ServerErrorCode.Bad_Request` code. @vgkholla do you think we could use that one here?

Reply:
@cgtz that's a good point - we certainly don't want to waste resources on that. I have a feeling that `BadRequest` might turn into an `InternalServerError` at the `Router` since the `Router` is not supposed to be making any bad requests. The other option is to make a new error code for this and then get the router to understand it. However, that means that the new router code should be deployed before this is (which might be ok). Perhaps we can do that in a follow up PR (immediately after). @zzmao what do you think?

Reply:
To be clear, I suggest we keep this code right now and change the mapping in the follow up PR

Reply:
Using Blob_Not_Found in this PR. Issue #860 is created for the following changes.